website/docs/components/form/file-input/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,31 @@
+## Usage
+
+### When to use
+
+- Use a file input to let users quickly select files from their computer.
+
+### Disabled State
+
+The File Input component allows users to upload documents and images. Disabling it can make users feel confused, frustrated, and unsure about why it’s not working.
+
+Instead of disabling, consider these alternatives:
+
+- Include explanatory text or prompts: If you display a disabled file input, explain why it’s disabled and what users need to do to enable it. You can use a tooltip, or an instructional sentence.
```

Comment:
Can you actually use a tooltip in this case (technically speaking)? Maybe @KristinLBradley has knowledge of this. Either way, I might recommend reordering the options to promote what we intend to be the best experience, which in this case I would think is maybe more like this:

- Helper text in the field component 
- A descriptive/instructional sentence
- Tooltip

From a UX perspective, the _appearance_ of a disabled button communicates that an element cannot be interacted with, therefore, a user may not think to hover over the element.

Reply:
+1 @jorytindall 
Since you can't focus on a disabled element, and the element that is disabled is also hidden to users with assistive tech, a tooltip wouldn't work.

Reply:
You could include a `Tooltip` in the label or in the helper text.